core/client/app/styles/patterns/forms.css

```diff
@@ -122,6 +122,7 @@ select {
     font-size: 1.4rem;
     font-weight: normal;
     user-select: text;
+    cursor: pointer;
```

Comment:
This adds `cursor: pointer` to all text fields not just select boxes, overriding the default text selection pointer - was this the intended effect?

/cc @JohnONolan